test/integration/client-side-operations-timeout/node_csot.test.ts

```diff
@@ -457,17 +457,17 @@ describe('CSOT driver tests', metadata, () => {
             const cursor = client
               .db('db')
               .collection('coll')
-              .find({}, { batchSize: 1, timeoutMode: 'iteration', timeoutMS: 100 })
+              .find({}, { batchSize: 1, timeoutMode: 'iteration', timeoutMS: 200 })
               .project({ _id: 0 });
             await cursor.toArray();
 
             expect(commandStarted).to.have.length.gte(3); // Find and 2 getMores
+
             expect(
               commandStarted.filter(ev => {
                 return (
-                  ev.command.find != null &&
-                  ev.command.getMore != null &&
-                  ev.command.maxTimeMS != null
+                  (ev.command.find != null && ev.command.maxTimeMS != null) ||
```

Comment:
I think the previous condition was impossible, because it wanted both a find and a getMore in the same command.  So it was always true.

Reply:
If it returns the right result this is fine, but still seems like a confusing condition. If you feel similarly maybe we should break this up, 

const finds = filter finds;
const getMores = filter getMores;
expect finds do not have maxTimeMS
expect getMores do not have maxTimeMS